front/src/common/redux/reducers/groupListReducer.js

```diff
@@ -0,0 +1,15 @@
+
+import * as types from '../actions/types';
+
+const initialState = {
+  currentGroup: 1,
```

Comment:
It is better to store in `state` actual group object with all of the data, not the id itself

Reply:
done